### PR TITLE
Doc: Add 'scope' section and illustration of PEtab components

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: doc/documentation_data_format.rst [hunk 1]
```diff
@@ -15,6 +15,27 @@ systems biology, especially for the case of Ordinary Differential Equation
...
+
+The scope of PEtab is the full specification of parameter estimation problems
+in typical systems biology applications. In our experience, a typical setup of
+data-based modeling starts either with (i) the model of a biological system
+that is to be calibrated, or with (ii) experimental data that are to be
+integrated and analyzed using a computational model.
+Measurements are linked to the biological model by an observation and noise
+model. Often, measurements are taken after some perturbations have been
+applied, which are modeled as derivations from a generic model
+(Subfigure A below). Therefore, one goal was to specify such a setup in the
```
Comment (yannikschaelte) [inline doc/documentation_data_format.rst:29]: This document is also auto-converted as part of the supplement, and in the pdf "below" may be wrong. is it possible to assign a figure index?

File: doc/documentation_data_format.rst [hunk 2]
```diff
@@ -15,6 +15,27 @@ systems biology, especially for the case of Ordinary Differential Equation
...
+least redundant way. Furthermore, we wanted to establish an intuitive, modular,
+machine- and human-readable and -writable format that makes use of existing
+standards.
+
+.. image:: gfx/petab_scope_and_files.png
+   :alt: A common setup for data-based modeling studies and it is represented in PEtab.
```
Comment (yannikschaelte) [inline doc/documentation_data_format.rst:35]:
```suggestion
   :alt: A common setup for data-based modeling studies as it is represented in PEtab.
```

**Review by LeonardSchmiester — APPROVED** [review]
looks good :+1:

File: doc/documentation_data_format.rst [hunk 3]
```diff
@@ -45,6 +66,9 @@ i.e.
 .. image:: gfx/petab_files.png
    :alt: Files constituting a PEtab problem
 
+Subfigure B above shows how those files relate to a common setup for
```
Comment (yannikschaelte) [inline doc/documentation_data_format.rst:69]: see above "above"

**Review by yannikschaelte — APPROVED** [review]

File: doc/documentation_data_format.rst (hunk 1 above)
Comment (dweindl) [reply doc/documentation_data_format.rst]: Changed.

**Review by yannikschaelte — APPROVED** [review]
cool

**Pull request merged**